MANIFEST.in

```diff
@@ -1,4 +1,6 @@
-recursive-include django_downloadview *
+recursive-include demo *.py *.rst *.html *.json *.txt
```

Comment:
The base from which the package is generated is a tarball. The is a tool called [`uscan`](https://wiki.debian.org/debian/watch) that is used to scan for new upstream releases. I could use the git repo but in the end I would still need it to generate a tarball that can can be uploaded to the Debian archive. Therefore having `python setup.py sdist` generate a complete tarball is usually the easiest way. Additionally if I would just clone your repo to create the tarball I would also have to take care about excluding files like `.gitignore` and the like that have no place in a proper source tarball.

The tests are run when someone (usually a Debian Developer or Debian Maintainer) builds the package from the source tarball. This is considered best-practice if tests are available from upstream. Plus, those tests, if included in the tarball and wired up in the Debian packaging code are run constantly by the [Debian CI](http://ci.debian.net/) to check for regression in the package or its dependencies.

If you want to learn about packaging for Debian, a good starting point is the [Debian Wiki](https://wiki.debian.org/Python/Packaging).